### PR TITLE
blog for auto-transformation of sample app from JEE8 to jakarta EE 9

### PR DESCRIPTION
**Pull request by m-brophy**

### CODE REVIEW

File: src/content/blog/javax-jakarta-openrewrite-automigrate/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+#Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
```
Comment (mrizzi) [inline src/content/blog/javax-jakarta-openrewrite-automigrate/index.md]: Please add the metadata section at the beginning of the file.
Something like
```suggestion
---
title: Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
description: Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
featured: true
author: m-brophy
date: 2022-08-17T00:00:00Z
tags:
  - docs
---
```
with:

- `title` being the title shown in the web page
- `description` is for SEO (so having it like `title` makes sense to me)
- `featured` makes it appear in the `Featured Posts` right menu in blog web page
- `author` is not yet used but we agreed on using the GH username
- `date` is set to the expected/planned release day

Another note we figured out yesterday is that `#` (single) is not allowed in the body of the blog post hence the "bigger" heading allowed is `##`

File: src/content/blog/javax-jakarta-openrewrite-automigrate/index.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+When modernizing applications a popular path is the transformation from one able to build and run on Java EE 8 to doing the same on Jakarta EE 9. Windup can help users discover the changes they need to make to their applications to achieve this, and using it in conjunction with OpenRewrite an auto-migration can be performed. This article will demonstrate an auto-migration on a sample application.
+
+##Build of Java EE 8 Application
```
Comment (mrizzi) [inline src/content/blog/javax-jakarta-openrewrite-automigrate/index.md]:
```suggestion
## Build of Java EE 8 Application
```
Try add a space between `##` and the headings to have them properly rendered also in the next headings.

File: src/content/blog/javax-jakarta-openrewrite-automigrate/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+![screenshot-7](./bootstrappingResults.png)
+*screenshot-7: Transform bootstrapping files results*
+
+If we now run
+
+`mvn clean package`
+
+from the root of the completeDuke source folder, the transformed source project should build correctly without any errors.
+
+#Deployment to Jakarta EE 9 Application Server
```
Comment (mrizzi) [inline src/content/blog/javax-jakarta-openrewrite-automigrate/index.md]: As above,  `# ` (single) is not allowed in the body of the blog post hence the "bigger" heading allowed is `##`

**Review by mrizzi — COMMENTED** [review]
I shared some knowledge I had from my experience in writing the release notes.
Hope it helps :+1:

**Comment by PhilipCattanach** [comment]
Thanks @m-brophy the changes look good. Happy to squash and merge.

**Pull request merged**